tests/mechanic/launcher_test.py

```diff
@@ -247,6 +248,44 @@ def test_bundled_jdk_not_in_path(self):
 
         # unmodified
         self.assertEqual(os.environ["PATH"], env["PATH"])
+        self.assertIsNone(env.get("JAVA_HOME"))
+
+    def test_pass_env_vars(self):
+        cfg = config.Config()
+        cfg.add(config.Scope.application, "mechanic", "keep.running", False)
+        cfg.add(config.Scope.application, "system", "env.name", "test")
+        cfg.add(config.Scope.application, "system", "passenv", "JAVA_HOME,FOO1")
+        os.environ["JAVA_HOME"] = "/path/to/java"
+        os.environ["FOO1"] = "BAR1"
+
+        proc_launcher = launcher.ProcessLauncher(cfg)
+
+        t = telemetry.Telemetry()
+        # no JAVA_HOME -> use the bundled JDK
```

Comment:
Note: this is with ES >= 7

Reply:
This comment is here because we are passing `java_home=None` in to the `_prepare_env` method and is not dependent on a version in the unit test

Reply:
Ok thanks for clarifying